app/javascript/application.js

```diff
@@ -1,37 +1,37 @@
 /* global window */
 
-import './jQueryGlobalizer.js'
-import 'bootstrap'
-import 'bootstrap-select'
-import './sweet-alert-confirm.js'
+import "./jQueryGlobalizer.js";
+import "bootstrap";
+import "bootstrap-select";
+import "./sweet-alert-confirm.js";
 
```

Comment:
Change all of the double quotes back. It's causing the [linter](https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/) to fail. Other things are causing the linter to fail too. Check the yarn lint check output for more details